### ISSUE TITLE
Document views behavior

### ISSUE THREAD

**Issue by imaia** [mattupstate/flask-security#587]
flask-security custom views lack documentation; which are they? How to use them? Specially if through json requests.

**Comment by jirikuncar** [comment]
@imaia do you have any proposal how to extend http://flask-security.readthedocs.io/en/latest/customizing.html#views ?

**Comment by imaia** [comment]
Maybe. I was thinking about a new section explaining which views flask-security provides and how to use them. Something like: view login; it has support for regular requests and json requests; the regular request behavior is transparent; with json, it should be used like this: <explanation>

There is already the customizing, explaining the templates, but there is nothing on the json requests.

**Comment by jirikuncar** [comment]
@imaia would you mind drafting a quick proposal?

**Comment by imaia** [comment]
@jirikuncar yeah, custom views link is quite complete, but it doesn't cover json interaction. Example: a valid login request with json answer would send id and authorization_token as response; If the same request was done a second time, the answer would be plain html, which is not expected; I'll try writing a quick proposal.

**Comment by italomaia** [comment]
Opinion https://github.com/mattupstate/flask-security/pull/590

**Comment by imaia** [comment]
Made this draft over there, explaining some views behavior on json calls.
